firestore/tests/system/test_system.py

```diff
@@ -633,16 +632,19 @@ def test_query_with_order_dot_key(client, cleanup):
         .limit(3)
         .stream()
     )
-    found_data = [{u'count': 30, u'wordcount': {u'page1': 130}},
-                  {u'count': 40, u'wordcount': {u'page1': 140}},
-                  {u'count': 50, u'wordcount': {u'page1': 150}}]
+    found_data = [
+        {u"count": 30, u"wordcount": {u"page1": 130}},
+        {u"count": 40, u"wordcount": {u"page1": 140}},
+        {u"count": 50, u"wordcount": {u"page1": 150}},
+    ]
     assert found_data == [snap.to_dict() for snap in found]
     bad_cursor = {"wordcount.page1": last_value}
```

Comment:
Since both pass with your change, "good" and "bad" aren't great labels, let's use `cursor_with_dotted_paths` here, and `cursor_with_nested_keys` above,

Reply:
I renamed `cursor` and `dot_key_cursor` in name in  **conflict resolved** commit